docs/management_api.yml

```diff
@@ -154,6 +154,53 @@ paths:
         500:
           $ref: "#/responses/InternalServerError"
 
+  /deployments/group/{name}:
+
+    post:
+      operationId: Create Deployment for a Group of Devices
+      tags:
+        - Management API
+      security:
+        - ManagementJWT: []
+      summary: Create a deployment for a group of devices
+      description: |
+        Deploy software to devices belonging to the specified group.
+
+        Artifact is auto assigned to the device from all available artifacts based
+        on artifact name and device type. Devices for which there are no compatible
+        artifacts to be installed are considered finished successfully as well as
+        receive status of `noartifact`. If there is no artifacts for the deployment,
+        deployment will not be created and the 422 Unprocessable Entity status code
+        will be returned.
+
+      parameters:
+        - name: name
+          in: path
+          description: Device group name.
+          required: true
+          type: string
+        - name: deployment
+          in: body
+          description: New deployment that needs to be created.
+          required: true
+          schema:
+            $ref: "#/definitions/NewDeploymentForGroup"
+      produces:
+        - application/json
+      responses:
+        201:
+          description: New deployment created.
+          headers:
+            Location:
+              description: URL of the newly created deployment.
+              type: string
+        400:
+          $ref: "#/responses/InvalidRequestError"
```

Comment:
Maybe we should start adding 401 responses here as well?
(I can see it's also not documented for the other endpoints either).

Reply:
You are right, they are missing, I've pushed a new commit to add them.